spec/scenarios/pay_in_advance_charges_spec.rb

```diff
@@ -374,6 +374,268 @@
         expect(fee.amount_cents).to eq(400)
       end
     end
+
+    context 'when there is matching group' do
+      let(:transaction_id) { SecureRandom.uuid + 'test' }
```

Comment:
Style/StringConcatenation: Prefer string interpolation to string concatenation. (https://rubystyle.guide#string-interpolation)